source/whats-new.txt

```diff
@@ -41,6 +41,13 @@ What's New in 4.8
   affect the behavior of your application, see
   :ref:`<java-breaking-changes-v4.8>`.
 
+.. important:: Deprecation Notice
+
+    The ``IterableCodec`` class and ``MapCodec`` class are deprecated. Instead,
+    use the ``CollectionCodec`` class and ``MapCodecV2`` class.
```

Comment:
Suggestion: The word "class" is being used a lot in this small section. We could reduce it for easier readability:

```suggestion
    The ``IterableCodec`` and ``MapCodec`` classes are deprecated. Instead,
    use ``CollectionCodec`` and ``MapCodecV2``.
```